pyproject.toml

```diff
@@ -51,3 +53,6 @@ unfixable = ["B", "SIM"]
 [tool.pytest.ini_options]
 DJANGO_SETTINGS_MODULE = "inertia.tests.settings"
 django_find_project = false
+
+[tool.basedpyright]
+typeCheckingMode = "standard"
```

Comment:
I'm quite partial to mypy instead of pyright. I think its more the standard, but anything that encourages typing I'm here for. 

I think it would be important to add a way to run the type checking too. Perhaps even in the CI. It could be added later too.

Reply:
Yeah, that is a good point. I used mypy before, but it was somehow very slow on a project I was working on and pyright was much faster. For a library, I agree that using the most established solution is probably best.

I'll change to mypy and update the CI as well. Probably it's better to follow how it's being done there as a way to run the type checking.